### DISCUSSION TITLE
Confidence interval for the Minimum Depth indicator

### DISCUSSION THREAD

**Discussion by IuriLeite** [kogalur/randomForestSRC#398] [Q&A]
Dear All,
I am carrying out analysis of Random Survival Forest using the package RSC. I used two indicators for selecting variables: the VIMP and the Minimum depth. I estimated the confidence intervals for VIMP, but I did not succeed in estimating the confidence intervals for the variables selected using the minimum depth. Could you please tell me if there is a way for doing this?
I would also like to know what is the bets way to show the accuracy of the model prediction as I have more than 8000 observations and only 104 cases (the proportion of censoring observations  is very high)? 
Best regards,
Iuri Leite

**Comment by ishwaran** [comment]
Confidence intervals for minimal depth are not available in the software.

Regarding your last question.  The package uses C-index to measure performance but if censoring rate is very high, you could consider using the Brier score which is more robust to high censoring rates than C-index.  Calculating this value is discussed in the following vignette (see Figure 2) and code.

 https://www.randomforestsrc.org/articles/survival.html

**Reply by IuriLeite** [reply]
Dear Professor Hemant Ishwaran,
thanks for the message. I have already started looking at the material you suggested.
I would take advantage of this opportunity to ask if there is a way to look at the contribution for the C-index or the Brier Score among those who experienced the event. Hope this question makes sense. As I mentioned before, only 104 events occurred among more than 8000 participants. 
Do you think I could be less restrictive when selecting variables through Vimp by using a significance level of 10%?
Thanks again.
Best regards,
Iuri Leite
